spec/features/user_can_view_cart_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+feature "cart" do
+  fixtures :categories
+  fixtures :items
+  fixtures :cart
+
+  let(:cart) { Category.find(1) }
+
+  before(:each) do
+    visit "/"
+    click_link 'Cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.